### DISCUSSION TITLE
Issue with discussions

### DISCUSSION THREAD

**Discussion by nriesco** [community/community#5228] [Discussions]
Hi, I'm not sure if this is the right place, but there seems to be an issue with "Discussions".

I started this discussion (https://github.com/ImageMagick/ImageMagick/discussions/4099) but I can't mark the answer. It seems to me that something is off with the tooltip/suggestion/tip that is being shown (see the image below). When clicking "Got it!" I get a 422 error (only visible while inspecting it).

The "mark as answer" button cannot be clicked.

Thanks!

![image](https://user-images.githubusercontent.com/1995514/130327369-a79cb7d8-5ea9-4a0d-8b76-0a23403b678b.png)

**Comment by kmk3** [comment]
@nriesco on [Aug 31](https://github.com/github/feedback/discussions/5228#discussion-3532733):

> Hi, I'm not sure if this is the right place, but there seems to be an issue
> with "Discussions".
> 
> I started this discussion
> ([ImageMagick/ImageMagick#4099](https://github.com/ImageMagick/ImageMagick/discussions/4099))
> but I can't mark the answer. It seems to me that something is off with the
> tooltip/suggestion/tip that is being shown (see the image below). When
> clicking "Got it!" I get a 422 error (only visible while inspecting it).
> 
> The "mark as answer" button cannot be clicked.

Looks like a duplicate of #5158.
